pom.xml

```
@@ -65,9 +65,9 @@
 
         <!-- Maven Plugins -->
         <maven.version>3.9.8</maven.version>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>17</maven.compiler.target>
-        <maven.compiler.release>17</maven.compiler.release>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <maven.compiler.release>8</maven.compiler.release>
```

No need to overwrite all of these, only `release` matters.
```suggestion
        <maven.compiler.release>8</maven.compiler.release>
```